pkg/deploy/deploy.go

```diff
@@ -78,7 +79,7 @@ func (o *deployHandler) ApplyKubernetes(kubernetes v1alpha2.Component) error {
 	}
 
 	// Deploy the actual Kubernetes component and error out if there's an issue.
-	log.Infof("\nDeploying Kubernetes %s: %s", u.GetKind(), u.GetName())
+	log.Sectionf("Deploying Kubernetes Component: %s", u.GetName())
```

Comment:
So, we're going to use log pkg in the business layer? I guess it's much easier to deal with here.